### PR TITLE
Drupal 9 and prep for Drupal 10

### PR DESCRIPTION
**Pull request by fjgarlin**
PR para actualizar a Drupal 9 y dejarlo todo preparado para Drupal 10.

Hemos cambiado ciertos valores que estaban en configuración a variables de entorno. Las siguientes tienen que estar en el servidor:
```
PAYPAL_ENVIRONMENT // 'live' or 'sandbox'
PAYPAL_LIVE_CLIENT_ID
PAYPAL_LIVE_CLIENT_SECRET
PAYPAL_SANDBOX_CLIENT_ID
PAYPAL_SANDBOX_CLIENT_SECRET

SWIFTMAILER_USERNAME // postmaster@mg.asociaciondrupal.es
SWIFTMAILER_PASSWORD
```

Cuando se despliegue el código (`composer install`), hay que ejecutar: `drush deploy` or `vendor/bin/drush deploy`.

### CODE REVIEW

**Comment by fjgarlin** [comment]
PR against the `drupal-9` branch whose PR is here https://github.com/AsociacionDrupalES/AED/pull/120

TODO:
- [ ] Fix paypal custom module to D10.
- [ ] Fix paypal functionality not working after D9 upgrade.
- [ ] Fix AED theme to D10.
- [ ] Update to D10.
- [ ] Move site to `web` folder.
- [ ] Move config to `config` folder.

File: sites/default/conf/google_analytics.settings.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+_core:
```
Comment (isholgueras) [inline sites/default/conf/google_analytics.settings.yml:1]: Google analytics should be disabled sooner or later. We're not checking GA and we're adding some tracking to the web.

File: sites/default/conf/core.extension.yml [hunk 2]
```diff
@@ -32,6 +31,7 @@ module:
   field_ui: 0
   file: 0
   filter: 0
+  google_analytics: 0
```
Comment (isholgueras) [inline sites/default/conf/core.extension.yml:34]: This should be disabled as we don't want to track, at least with google analytics.

**Review by isholgueras — CHANGES_REQUESTED** [review]

File: sites/default/conf/core.extension.yml (hunk 2 above)
Comment (fjgarlin) [reply sites/default/conf/core.extension.yml]: Done.

**Comment by fjgarlin** [comment]
@isholgueras - the PR, as it is right now, it's a step further from your `drupal-9` update. I updated a few modules and settings and also made some modules D10 ready and brought core to the latest 9.5.X version.

It should be safe for you to merge to your `drupal-9` branch, then run `ddev composer install` and `ddev drush deploy` and then test the changes locally.

The only blocker for a D9 update seems to be the paypal integration which you mentioned it was broken. I will look into that first before going any further with D10. 

Roadmap could be:
- D9 update with Paypal working as expected
- Re-structure folders (`web`, `config`...)
- D10

**Pull request merged**